doc/devel/coding_guide.rst

```diff
@@ -1,20 +1,76 @@
-.. _reviewers-guide:
+.. _pr-guidelines:
 
-********************
-Reviewers guideline
-********************
+***********************
+Pull request guidelines
+***********************
 
-.. _pull-request-checklist:
+Pull requests (PRs) are the mechanism for contributing to Matplotlibs code and
+documentation.
 
-Pull request checklist
-======================
+.. _pr-checklist-authors:
 
-Branch selection
-----------------
+Checklist for PR authors
```

Comment:
I'd call this guidelines since these aren't really things you can check off

Reply:
That is correct. OTOH, I would like to keep a certain similarity between the names of the summary sections for authors and for reviewers.

Using "Guidelines for PR authors/reviewers" feels a bit too broad. Semantically I would like to express "Summary of things to do/check as a PR author/reviewer". I've now changed both to a shorted "Summary for PR authors/reviewers". Do you think that's suitable?